mk

```diff
@@ -671,17 +489,15 @@ OONIPRIVATE_REPO = git@github.com:ooni/probe-private
 
 # $(OONIPRIVATE) clones the private repository in $(GIT_CLONE_DIR)
 $(OONIPRIVATE): search/for/git $(GIT_CLONE_DIR)
-	test -d $(OONIPRIVATE) || $(MAKE) -f mk __really_clone_private_repo
-
-__really_clone_private_repo:
+	test -d $(OONIPRIVATE) && rm -rf $(OONIPRIVATE)
```

Comment:
This should be changed to become unconditional removal. We don't exercise this code path in cloud builds but it's nonetheless more robust to use `rm -rf` if this is what we want (and it is).